### PR TITLE
fix possible IndexError

### PR DESCRIPTION
**Pull request by dbouchillon**
Fixes ZEN-22369

### CODE REVIEW

File: ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/OperatingSystem.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def process(self, device, results, log):
             pass
 
         # if NTDS service present then this is a DC
-        if domainController:
+        if len(domainController):
```
Comment (joshw) [inline ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/OperatingSystem.py]: Technically don't really need to change this..  if domainController would do the same thing, i believe.

Comment (ssOleg) [inline ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/OperatingSystem.py]: len((None,)) == True
We get true in all cases.

Comment (byshovets) [inline ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/OperatingSystem.py]: So line 60 can be changed to: domainController = results.get('ActiveDirectory', ())

Comment (dbouchillon) [inline ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/OperatingSystem.py]: if there are no results returned, domainController results in an empty list, []
so len([]) == 0

i made another change to make this less confusing

**Pull request merged**